app/utils/decorator_utils.py

```diff
@@ -0,0 +1,33 @@
+from app.database.models.user import UserModel
+
```

Comment:
Why is there so much whitespace?

Reply:
@ramitsawhney27  I can remove it but removing it give rise to error on flake8.... What should I do?
![1](https://user-images.githubusercontent.com/40512667/52716606-235ef800-2fc5-11e9-92b8-65510ca405ea.png)

Reply:
I meant in the file in general, not this particular line.

Reply:
@ramitsawhney27 If I check it with pylint and get a score above 8.5..... is it okay?

Reply:
No, the code has to be up to the standards that mentorship-backend has, which have been indicated with the review comments you've received.